recipe/meta.yaml

```diff
@@ -8,7 +8,27 @@ package:
 source:
   - folder: .
     url: https://github.com/DOI-USGS/{{ name }}/archive/{{ version }}.tar.gz
-    sha256: a8ae036caa0b958fa6e28ee8ba7a29c2b6fefeb1413144243d59521f51d1dc12
+    sha256: 2f2b49fc74b6f2eef92839737315be7585b81331d78592bffdf04e31611e8eaf
+
+  # submodules
+  - folder: ale
+    url: https://github.com/DOI-USGS/ale/archive/0.10.0.zip
+    sha256: 2373e0e1ff3715adef60e61db22e6cea1d5b7941f5b1999dd202065d3d84efb2
+  - folder: csm
+    url: https://github.com/USGS-Astrogeology/csm/archive/c78dbba3731432249a75db9fdd7415b292d46715.zip
+    sha256: e3eb35fbc27df94d6432c66a93b1dd140a9e93d6897c9107d5a99ab0557e99ea
+  - folder: gtest
+    url: https://github.com/google/googletest/archive/aee0f9d9b5b87796ee8a0ab26b7587ec30e8858e.zip
+    sha256: 04a1751f94244307cebe695a69cc945f9387a80b0ef1af21394a490697c5c895
+  - folder: json
+    url: https://github.com/nlohmann/json/archive/refs/tags/v3.8.0.zip
+    sha256: 83947cb78d50990b4b931b8dbc8632781bc601baa45b75ece0899c7b98d86c0b
+  - folder: ale/eigen
+    url: https://gitlab.com/libeigen/eigen/-/archive/3.4.0/eigen-3.4.0.zip
+    sha256: 1ccaabbfe870f60af3d6a519c53e09f3dcf630207321dffa553564a8e75c4fc8
+  - folder: PROJ
+    url: https://github.com/OSGeo/PROJ/archive/9.3.0.zip
```

Comment:
Current PROJ is 9.3.1. What is the plan for an update cadence?

Reply:
Since the dependency will be built into the library I think an as needed cadence would be best. This is also true for some of the above dependencies like ALE and Eigen